components/RouteForm/RouteForm.js

```diff
@@ -51,14 +51,13 @@ export default function RouteForm({
             alt="Preview of the image to upload"
           />
         )}
-        <Label htmlFor="Image">Upload your image</Label>
-        <FileInput onChange={onChange} type="file" id="image" name="image" />
+        <FileInput labelText="Upload your image" onChangeInput={onChange} />
       </Fieldset>
       <ButtonContainer>
         <FormButton
-          disabled={sendRouteForm}
-          onClick={onCancelClick}
           type="button"
+          onClick={onCancelClick}
+          disabled={sendRouteForm}
```

Comment:
this looks oddly named 🤔 

is sendRouteForm a boolean? If so, the naming convention is like this:
isOpen, isClickable etc.